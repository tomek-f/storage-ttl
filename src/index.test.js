import * as storage from '.';

test('snapshot', () => {
  expect(storage).toMatchSnapshot();
});
