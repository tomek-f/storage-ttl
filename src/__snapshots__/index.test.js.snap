// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Object {
  "default": Object {
    "clear": [Function],
    "getItem": [Function],
    "getJson": [Function],
    "getJsonWithTtl": [Function],
    "key": [Function],
    "length": 0,
    "name": "localStorage++",
    "originalStorage": Storage {},
    "removeItem": [Function],
    "setItem": [Function],
    "setJson": [Function],
    "setJsonWithTtl": [Function],
    "support": true,
  },
  "localStorageSupport": true,
  "sessionStorage": Object {
    "clear": [Function],
    "getItem": [Function],
    "getJson": [Function],
    "getJsonWithTtl": [Function],
    "key": [Function],
    "length": 0,
    "name": "sessionStorage++",
    "originalStorage": Storage {},
    "removeItem": [Function],
    "setItem": [Function],
    "setJson": [Function],
    "setJsonWithTtl": [Function],
    "support": true,
  },
  "sessionStorageSupport": true,
  "storageSupport": true,
}
`;
